### DISCUSSION TITLE
2024-05-08-sony

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#187] [General]
# 我居然用回有线耳机了 - 索尼 MV1 - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.Life男人的快乐，我居然用回有线耳机了，最近周末抽时间去线下试了不少好耳机，最后很意外买了索尼这款 MV1 开放式监听耳机，也不降噪还有线，却出奇的喜欢，想和大伙聊聊我的选择过程。

https://tw93.fun/2024-05-08/sony.html

**Comment by todjiang** [comment]
头戴式耳机最大的问题， 戴不久。 时间长了， 太热了。。

**Reply by tw93** [reply]
哈哈 仅限夏天空调房

**Reply by yibie** [reply]
我用的一个无线耳机，音质也能达到这种监听耳机的水平，不过本来厂商也是发烧耳机的一个品牌，叫歌德。一个搞笑的事情值得分享，就是我这个耳机啊，无线时音质比有线时好太多了，我到现在都不明白为啥，但音质确实很不错来着，交响乐、人声都很适合

**Comment by hxz393** [comment]
头戴耳机还有一大问题....压头发造成头顶秃，后面换有线耳塞了，等啥时候全秃了再戴回耳机吧

**Reply by tw93** [reply]
哈哈哈哈

**Comment by Oosl** [comment]
之前闲鱼￥520捡了一个WH-1000XM4，但是佩戴有两个问题：  
1. 空调房戴都很热，闷出汗  
2. 戴眼镜再戴耳机，最多戴两小时就会头疼，夹得太紧了（可能单纯我头大？）  

木耳现在觉得OWS耳机很不错可以整天挂在耳朵上也不影响正常交流🤣

**Reply by tw93** [reply]
我这一款还比较舒服

**Comment by loserc6** [comment]
连接线是type c to c吗？软线吗，一直在找一款合适的c to c软线给耳机用，现阶段买的都不合适 太硬

**Reply by tw93** [reply]
定制了一个
不是软线
